src/_locales/fr/messages.json

```diff
@@ -153,10 +168,56 @@
     "hash": "9a20059d2d16dfb7e731226187c04836"
   },
   "optionMastodonHandleHelper": {
-    "message": "Veuillez entrer votre lien Mastodon dans le format « utilisateur@domaineserveur »",
+    "message": "Veuillez entrer votre lien Mastodon au format « utilisateur@domaineserveur »",
     "description": "An explanatory text shown to the user in the settings to explain what format they have to enter the Mastodon handle. It should not hint to any example (that's what the placeholder is for), but explain the general format.",
     "hash": "5e2a6e34c5fed46740dc238f1cc7768a"
   },
+  "optionRedirectInMainWindow": {
+    "message": "Bloque l’usage de pop-up.",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "dc056fc373c7161104ad81b28884a017"
+  },
+  "optionRedirectInMainWindowDescr": {
+    "message": "Bloque l’usage d’une fenêtre supplémentaire et redirige l’action sur la fenêtre principale.",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "ba0f60f1ae0c6d6785d5eb410247eb11"
+  },
+  "translatorCredit": {
+    "message": "Cette extension a été traduite en français par $TRANSLATORS$",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
```

Comment:
Note you should put your own name here (and above the link to your GitHub profile or so). Or actually, as not only you translated this and the others also forget, [please follow our guidelines on how to credit all translators](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated and how the values get in there, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Reply:
Ha, figured it was strange.

Reply:
Actually, if you want, it would be a great idea [if you could try out the development version](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) and report back whether it/everything works/you like it or what may still be adjusted, so I can do this before the next release. In case there is something to be done… :smile: 

Just so we know the translation works good in the browser when it is actually used.

Reply:
Well, that may be a bit out of my comfort zone to do so. But I’ll try my best to do so

Reply:
Following the instructions I fail to get the submodules, especially the CommonCss which ask me for authentification.